From 5fb4ebec8d58d27cfc931da9ecd3ad35b26d25fc Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Thu, 25 Jun 2015 15:45:37 +0800
Subject: [PATCH] MLK-11168-1 ARM: dtsi: imx6ul: add iram property to support
 internal SRAM

By doing this, we can allow SDMA driver to allocate its memory from
iram when using i.MX6UL SoC.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
---
 arch/arm/boot/dts/imx6ul.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6ul.dtsi b/arch/arm/boot/dts/imx6ul.dtsi
index aaffb4b..8b451f5 100644
--- a/arch/arm/boot/dts/imx6ul.dtsi
+++ b/arch/arm/boot/dts/imx6ul.dtsi
@@ -730,6 +730,7 @@
 					 <&clks IMX6UL_CLK_SDMA>;
 				clock-names = "ipg", "ahb";
 				#dma-cells = <3>;
+				iram = <&ocram>;
 				fsl,sdma-ram-script-name = "imx/sdma/sdma-imx6q.bin";
 			};
 
-- 
1.8.0

